pkg/azureutils/azure_disk_utils.go

```diff
@@ -194,6 +184,39 @@ func GetCloudProvider(kubeconfig, secretName, secretNamespace, userAgent string)
 	return az, nil
 }
 
+// GetCloudProviderFromConfig get Azure Cloud Provider
+func GetCloudProvider(kubeConfig, secretName, secretNamespace, userAgent string) (*azure.Cloud, error) {
+	kubeClient, err := GetKubeClient(kubeConfig)
+	if err != nil && !os.IsNotExist(err) && err != rest.ErrNotInCluster {
+		return nil, fmt.Errorf("failed to get KubeClient: %v", err)
+	}
+	az, err := GetCloudProviderFromClient(kubeClient, secretName, secretNamespace, userAgent)
+	return az, err
+}
+
+// GetKubeConfig gets config object from config file
+func GetKubeConfig(kubeconfig string) (config *rest.Config, err error) {
```

Comment:
is this func `GetKubeConfig` needed? I thought only `GetKubeClient` is needed.

Reply:
We need this function in our V2 implementation.